src/main/java/LottoRank.java

```diff
@@ -0,0 +1,41 @@
+public enum LottoRank {
+    FIRST(1, 6, false, 2000000000),
+    SECOND(2, 5, true, 30000000),
+    THIRD(3, 5, false, 1500000),
+    FOURTH(4, 4, false, 50000),
+    FIFTH(5, 3, false, 5000),
+    FAIL(6, 0, false, 0);
```

Comment:
```suggestion
public enum LottoRank {
    FIRST(1, 6, false, 2_000_000_000),
    SECOND(2, 5, true, 30_000_000),
    THIRD(3, 5, false, 1_500_000),
    FOURTH(4, 4, false, 50_000),
    FIFTH(5, 3, false, 5_000),
    FAIL(6, 0, false, 0);
```
3자리씩 끊어주면 가독성이 높아지겠네요! 

Cost라는 클래스가 있는데 당첨금은 원시값을 그대로 사용한 이유가 있을까요? 어떤 기준으로 클래스로 포장하고 원시값을 그대로 사용하나요? 

> 규칙 3: 모든 원시값과 문자열을 포장한다. 

모두 포장하기는 어렵겠지만 최대한 포장해 보는 것은 어떨까요?

Money라는 클래스가 있으면 당첨금, Cost 모두 변수명으로 나타낼 수 있지 않을까요?